cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/reserve_transfer.rs

```diff
@@ -1466,3 +1474,334 @@ fn reserve_withdraw_from_untrusted_reserve_fails() {
 		assert!(result.is_err());
 	});
 }
+
+#[test]
+fn withdraw_and_deposit_erc20s() {
```

Comment:
all of the tests here are not cross-chain tests - they could also go to the runtime's unit-tests

if you want to keep them under emulated cross-chain tests, at least move them to some other file, this one is getting huge and there are not really reserve transfers.